src/Phinx/Config/Config.php

```diff
@@ -154,6 +155,12 @@ public function getEnvironment($name)
                 $environments[$name]['default_migration_table'] =
                     $this->values['environments']['default_migration_table'];
             }
+			// inject any other custom $config entries outside of the $config['environments'] entry
+			foreach ($this->values as $key => $value) {
+				if ($key !== self::CONFIG_ENVIRONMENTS) {
+					$environments[$name][$key] = $this->values[$key];
```

Comment:
so, If I understand, any configuration value that is not "environments" is global to all the environments?